centaur/src/main/resources/standardTestCases/draft3_read_write_functions_windows_line_endings.test

```diff
@@ -0,0 +1,26 @@
+name: draft3_read_functions_windows_line_endings
```

Comment:
Per our earlier meeting I'm not sure we need all of these variants of WDL tested, but I'll defer to @cjllanwarne on which ones we do want.

Reply:
Per @aednichols Cromwell currently supports draft_2 and draft_3 WDL versions and biscayne is currently in development. So I think it's better to test them all to be on a safe side.

Reply:
Realistically, the `1.0` and `development` versions of WDL are probably similar enough in how they use IO that I think we can get away with only testing one or the other of them. WDL draft-2 is "supported" but we're not explicitly adding new functionality to it (even if occasionally it benefits from work being done in 1.0, like here), so it would probably also be valid to omit the draft-2 tests too.

So: if it were me, I'd probably write a single WDL 1.0 test case for this, but I don't think it's _invalid_ to have the other versions tested too.

Reply:
Removed wdl-2 and wdl-3 tests, kept biscayne.